### ISSUE TITLE
Wavelet transform

### ISSUE THREAD

**Issue by Ystartff** [Yanfeng-Zhou/XNet#4]
HI! Congratulations on your successful submission,I checked your xnet.py and found that there are no wavelets in it, but I found that they are provided in your wavelet2D.py, is your wavelet a pre-processing method? After processing, save as two paths and enter them separately? If I misunderstand, would you please provide the corresponding code

**Comment by Yanfeng-Zhou** [comment]
You are right. Our model uses low-frequency and high-frequency images produced by wavelet transform as input. So you can prepare it before training!

**Comment by Ystartff** [comment]
I've noticed that using your wwavelet2D causes the image width to be halved, is that how your input works?
